### PR TITLE
save OVS Flow TEID to register in pipelined #6596

### PR DESCRIPTION
**Pull request by prabinakpattnaik**
Signed-off-by: prabina pattnaik <prabinak@wavelabs.ai>

## Summary

Add reg9 for store TEID value in ovs table 0 

## Test Plan
Output of table 0 entries:
vagrant@magma-dev:~/magma/lte/gateway$ sudo ovs-ofctl -O OpenFlow13 dump-flows gtp_br0 table=0
 cookie=0x0, duration=1.903s, table=0, n_packets=0, n_bytes=0, priority=10,tun_id=0x5,in_port=gtp0 actions=set_field:02:00:00:00:00:01->eth_src,set_field:ff:ff:ff:ff:ff:ff->eth_dst,set_field:0x5->reg9,set_field:0x1388->metadata,resubmit(,1)
 cookie=0x0, duration=1.891s, table=0, n_packets=0, n_bytes=0, priority=10,ip,in_port=LOCAL,nw_dst=192.168.128.30 actions=set_field:0x3e8->tun_id,set_field:192.168.60.141->tun_dst,set_field:0x8000->reg8,set_field:oam->tun_flags,set_field:0x5->reg9,set_field:0x1388->metadata,resubmit(,1)
 cookie=0x0, duration=1.891s, table=0, n_packets=0, n_bytes=0, priority=10,ip,in_port=mtr0,nw_dst=192.168.128.30 actions=set_field:0x3e8->tun_id,set_field:192.168.60.141->tun_dst,set_field:0x8000->reg8,set_field:oam->tun_flags,set_field:0x5->reg9,set_field:0x1388->metadata,resubmit(,1)
 cookie=0x0, duration=1.891s, table=0, n_packets=0, n_bytes=0, priority=10,arp,in_port=LOCAL,arp_tpa=192.168.128.30 actions=set_field:0x1388->metadata,resubmit(,1)
 cookie=0x0, duration=1.891s, table=0, n_packets=0, n_bytes=0, priority=10,arp,in_port=mtr0,arp_tpa=192.168.128.30 actions=set_field:0x1388->metadata,resubmit(,1)
 cookie=0x0, duration=1.903s, table=0, n_packets=69, n_bytes=5006, priority=0 actions=resubmit(,1)
 cookie=0x0, duration=1.903s, table=0, n_packets=0, n_bytes=0, priority=0,in_port=ipfix0 actions=resubmit(,201)
 cookie=0x0, duration=1.903s, table=0, n_packets=0, n_bytes=0, priority=0,in_port="li_port" actions=resubmit(,202)

### CODE REVIEW

File: lte/gateway/python/magma/pipelined/app/classifier.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 from magma.pipelined.app.inout import INGRESS
 from magma.pipelined.openflow import flows, messages
 from magma.pipelined.openflow.magma_match import MagmaMatch
-from magma.pipelined.openflow.registers import TUN_PORT_REG, Direction
+from magma.pipelined.openflow.registers import TUN_PORT_REG, TUN_ID_REG, Direction
```
Comment (pshelar) [inline lte/gateway/python/magma/pipelined/app/classifier.py]: Can you rename this that tells its ingress tunnel id.

**Review by pshelar — CHANGES_REQUESTED** [review]

File: lte/gateway/python/magma/pipelined/app/classifier.py (hunk 1 above)
Comment (prabinakpattnaik) [reply lte/gateway/python/magma/pipelined/app/classifier.py]: changed the name as "INGRESS_TUN_ID_REG".

File: lte/gateway/python/magma/pipelined/app/classifier.py [hunk 2]
```diff
@@ -162,6 +164,10 @@ def initialize_on_connect(self, datapath):
         self._install_default_tunnel_flows()
         self._install_internal_pkt_fwd_flow()
         self._install_internal_conntrack_flow()
+       # ue_ip_addr = "192.168.128.30"
+       # self.add_tunnel_flows(65525, 5, 1000,
+       #               IPAddress(version=IPAddress.IPV4,address=ue_ip_addr.encode('utf-8')),
+       #               "192.168.60.141", 5000, True)
```
Comment (pshelar) [inline lte/gateway/python/magma/pipelined/app/classifier.py]: can you also fix the lint issues posted on this review.

**Review by pshelar — CHANGES_REQUESTED** [review]

File: lte/gateway/python/magma/pipelined/app/classifier.py (hunk 2 above)
Comment (prabinakpattnaik) [reply lte/gateway/python/magma/pipelined/app/classifier.py]: Sorry, I forgot to remove the commented code for UT purpose. I have removed it.

**Review by koolzz — APPROVED** [review]
cc @pshelar

**Review by pshelar — APPROVED** [review]

**Pull request merged**